components/ai_chat/resources/page/components/feedback_form/index.tsx

```diff
@@ -86,6 +92,11 @@ function FeedbackForm(props: FeedbackFormProps) {
             />
           </label>
         </fieldset>
+        <fieldset>
+            <Checkbox checked={shouldSendUrl} onChange={handleCheckboxChange}>
+              <label>Share the URL of the page you are on (example.com) to help us make Leo better.</label>
```

Comment:
Here we should be displaying the real domain that will be sent to our backend instead of `example.com` literally.

Reply:
oops, changed and localized the string too